addons/xeh/CfgVehicles.hpp

```diff
@@ -397,4 +397,9 @@ class CfgVehicles {
     class Land_MapBoard_01_Map_Syndikat_Tanoa_F: Land_Map_blank_F {
         XEH_ENABLED;
     };
+
+    // AOW (2.02)
+    class GalleryDioramaUnit_01_base_F: Items_base_F {
+        XEH_ENABLED;
+    };
```

Comment:
This change does not necessarily require 2.02, but I guess it is a good thing to update this anyway? Would merge the PR now if the requirement was not set.

Reply:
This is based on current RC and sometimes things change, so we shouldn't merge until 2.02 is officially released

Reply:
still needed in release